### ISSUE TITLE
Suche unklar

### ISSUE THREAD

**Issue by usauerbrey** [jansauerbrey/recipe-app#12]
Wenn man z.B. "my recipies" auswählt, wird diese Auswahl nirgends mehr angezeigt. D.h. man weiß nicht, dass man eine Selektion getroffen hat.
schlecht
Wenn man nun eine Auswahl in den Tags trifft oder nach einem Text sucht, geschieht das immer nur in der vorherigen Auswahl. Die man nicht angezeigt bekommt und die man auch nicht mehr verändern kann. D.h alles von vorne neu machen ... 
ganz schlecht

**Comment by jansauerbrey** [comment]
Ja, ich werd die gesamte listung der Rezepte überarbeiten und aus jedem Teil einen eigenen "state" machen.
